app/src/main/res/layout/fragment_list_of_photos.xml

```diff
@@ -1,8 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.recyclerview.widget.RecyclerView
         xmlns:android="http://schemas.android.com/apk/res/android"
-        android:id="@+id/photos"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:clipToPadding="false"
-        android:padding="16dp" />
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/photos"
+    android:layout_width="0dp"
+    android:layout_height="0dp"
+    android:padding="6dp"
+    android:clipToPadding="false"
+    app:layoutManager="androidx.recyclerview.widget.GridLayoutManager"
+    app:layout_constraintBottom_toBottomOf="parent"
+    app:layout_constraintLeft_toLeftOf="parent"
+    app:layout_constraintRight_toRightOf="parent"
+    app:layout_constraintTop_toTopOf="parent" />
```

Comment:
Actually we don't have to specify these parameters here. They are from `ConstraintLayout`, but our RecyclerView is not in ConstraintLayout (by default it's FrameLayout) so setting width and height to `match_parent` is enough.